src/queryparser.py

```diff
@@ -0,0 +1,16 @@
+
+def parse(filter: str, varList: list):
```

Comment:
`varList` should be `var_list` (snake_case) to be consistent with coding style.

Reply:
Thank you, it was fixed